ui/admin/app/templates/scopes/scope/targets/target/manage-alias.hbs

```diff
@@ -37,7 +37,7 @@
     <page.actions>
       {{#if (or perms.canUpdate perms.canDelete)}}
 
-        <Hds::Dropdown as |dd|>
+        <Hds::Dropdown data-test-manage-alias as |dd|>
```

Comment:
Do we want to specify that this is the manage alias dropdown in the targets route?

Reply:
ah great point!